src/gui/src/drcWidget.cpp

```diff
@@ -765,27 +765,54 @@ void DRCWidget::loadJSONReport(const QString& filename)
                    filename.toStdString(),
                    e1.what());
   }
-
+  auto tech = block_->getDataBase()->getTech();
+  int dbUnits = block_->getDataBase()->getTech()->getDbUnitsPerMicron();
+  // check if DRC key exists
+  if (tree.find("DRC") == tree.not_found()) {
+    logger_->error(utl::GUI,
+                   87,
+                   "Unable to find the DRC key in JSON file {}",
+                   filename.toStdString());
+  }
```

Comment:
I'm not sure we would want to limit ourselves to just a DRC root. I think you could look at any root?

Reply:
When we have another possible type it seems easy to add it.  I agree that we won't always have DRCs.  I suspect one file will always be of one type so multiple roots don't see needed.

Reply:
@maliberty yes, I doubt we would combine files, I meant we could take the first root and more or less ignore the name of it, until we support more than DRC in which case it could be used to call `processDRC` or something like that.

Reply:
I think it better to show we don't support a new syntax than hope we can parse it.  We could make it a warning if the root isn't DRC.